packages/prisma/schema.prisma

```diff
@@ -438,6 +438,9 @@ model Team {
   attributes             Attribute[]
   smsLockReviewedByAdmin Boolean                 @default(false)
 
+  /// @zod.custom(imports.intervalLimitsType)
+  bookingLimits Json?
```

Comment:
Is a JSON field the best approach here? What other alternatives were considered?

Reply:
Event-type booking limits are also using a JSON field. I decided to take the same approach here to reuse as much logic as possible.

Reply:
yes, I agree that we should move away from JSON field for booking limits but I think this is a separate task. I will create an issue for it